docs/admin-guide/audit/audit.md

```diff
@@ -150,3 +156,26 @@ nodes:
  
     ...
 </pre>
+
+### vSphere Clusters
```

Comment:
I think this section is probably more relevant in a troubleshooting doc

Reply:
Agreed. Removed in 711b01c